### PR TITLE
Feature/dynamic opens

### PR DESCRIPTION
**Pull request by afek854**
## Overview

Note: Maybe in sensitive file we don't want it.

### CODE REVIEW

File: pkg/ruleengine/v1/helpers.go [hunk 1]
```diff
@@ -153,3 +153,32 @@ func interfaceToStringSlice(val interface{}) ([]string, bool) {
 	}
 	return nil, false
 }
+
+func CompareDynamic(dynamicPath, regularPath string) (string, bool) {
+	const dynamic = "<dynamic>"
```
Comment (matthyx) [inline pkg/ruleengine/v1/helpers.go:158]: this should use the same const as the storage, otherwise we might get into trouble

**Review by slashben — DISMISSED** [review]

File: go.mod [hunk 2]
```diff
@@ -22,7 +22,7 @@ require (
 	github.com/kubescape/backend v0.0.20
 	github.com/kubescape/go-logger v0.0.22
 	github.com/kubescape/k8s-interface v0.0.170
-	github.com/kubescape/storage v0.0.111
+	github.com/kubescape/storage v0.0.112-0.20240912075751-9a554eb26977
```
Comment (matthyx) [inline go.mod:25]: use the release pls

**Review by matthyx — CHANGES_REQUESTED** [review]
update storage

File: pkg/ruleengine/v1/helpers.go (hunk 1 above)
Comment (afek854) [reply pkg/ruleengine/v1/helpers.go]: DONE

File: go.mod (hunk 2 above)
Comment (afek854) [reply go.mod]: I need to merge https://github.com/kubescape/storage/pull/148
before

**Pull request merged**